bedrock/settings/base.py

```diff
@@ -1166,11 +1167,13 @@ def get_default_gateway_linux():
     "data.track.convertexperiments.com",
     "1003350.track.convertexperiments.com",
     "1003343.track.convertexperiments.com",
+    "cdn.cookielaw.org",
 ]
 CSP_STYLE_SRC = CSP_DEFAULT_SRC + [
     # TODO fix things so that we don't need this
     "'unsafe-inline'",
     "app.convert.com",
+    "cdn.cookielaw.org",
```

Comment:
Do we need this? I'm not seeing any styles coming from this domain but I could be missing something.

Reply:
Same as below, Thanks craig!